docker-compose.yaml

```diff
@@ -27,12 +36,17 @@ services:
     depends_on:
       - graphdb
     ports:
-      - 8900:8900
+      - "8900:8900"
     environment:
       - PSK
-      - RESOTOCORE_GRAPHDB_SERVER=http://graphdb:8529
-      - RESOTOCORE_GRAPHDB_PASSWORD=changeme
     command:
+      - --graphdb-server
+      - http://graphdb:8529
```

Comment:
Should we switch this to args?
a) the password will be in plain text in the process list of the host
b) the password should come from something like a k8s secret or vault and will then be handed in via the env.
By switching from env to cli flags we're teaching a bad behaviour.

Reply:
You are right - changed back to env var.